misc/repeater.talon

```diff
@@ -1,5 +1,5 @@
-<user.ordinals>: core.repeat_command(ordinals)
-
-
-
-
+# -1 because we are repeating, so the initial command counts as one
+<user.ordinals>: core.repeat_command(ordinals-1)
+repeat that: core.repeat_command(1)
+twice: core.repeat_command(1)
+# add N times
```

Comment:
maybe add this for n-times?
repeat that <number>: core.repeat_command(number)

also fyi you can do (in case you didn't know)
(repeat that | twice): core.repeat_command(1) 
to alias commands, though i *might* remove twice for my repro to avoid too many aliases for newbies